controller/internal/policy/api_session_enforcer.go

```diff
@@ -91,11 +100,13 @@ func (s *ApiSessionEnforcer) Run() error {
 		if err = s.appEnv.GetHandlers().ApiSession.DeleteBatch(ids); err != nil {
 			logrus.WithError(err).Error("failure while batch deleting expired api sessions")
 
+			s.appEnv.GetMetricsRegistry().Gauge(ApiSessionEnforcerDelete).Update(int64(len(ids)))
 			for _, id := range ids {
 				if err = s.appEnv.GetHandlers().ApiSession.Delete(id); err != nil {
 					logrus.WithError(err).Errorf("failure while deleting expired api session: %v", id)
 				}
 			}
+			s.appEnv.GetMetricsRegistry().Gauge(ApiSessionEnforcerDelete).Update(0)
```

Comment:
Not sure if this is the right kind of metric (gauge) for this type of quick updating.

Not sure if it won't just always show as 0 in the output.

Reply:
Will use meter instead